templates/etc_init_archivematica-storage-service.conf.j2

```diff
@@ -5,4 +5,8 @@ author "Artefactual Systems"
 start on (runlevel [2345] and filesystem)
 stop on runlevel [016]
 
+env LANG="en_US.UTF-8"
+env LC_ALL="en_US.UTF-8"
+env LC_LANG="en_US.UTF-8"
```

Comment:
You know, I think I've figured why. gunicorn is written in python too and the gunicorn workers are just spawned processes created with [`os.fork()`](https://github.com/benoitc/gunicorn/blob/2d11e5c7cc8e79ab2661d3fe50dd7d165834115f/gunicorn/arbiter.py#L546). Each worker is a new process but it's basically a copy of the previous instance. Later, [the worker loads the environment](https://github.com/benoitc/gunicorn/blob/2b839ca14437c61780d6eaaa8b24ad31e021f2c6/gunicorn/workers/base.py#L100-L103) that we define in the configuration file, but at that point the Python interpreter has already been bootstrapped so the locale settings are not taking into account.

One possible alternative: should we maybe try to make this change from the application code using the [locale module](https://docs.python.org/2/library/locale.html)? I'm not sure if it's possible but it would be preferably because this solution feels like it's solving the problem in the wrong layer, i.e. configuration. To be specific, I think the locale settings should be set here: https://github.com/artefactual/archivematica/blob/qa/1.x/src/dashboard/src/apache/django.wsgi#L39. I have no idea if this is going to work but it may be worth giving it a shot. Otherwise your solution is good enough, but please remember that RPM and DEB packages maintain its own init services definitions and they'll need to be updated too!